libs/blocks/gnav/gnav.js

```diff
@@ -520,7 +520,7 @@ class Gnav {
 
   setBreadcrumbSEO = () => {
     const seoEnabled = getMetadata('breadcrumb-seo') !== 'off';
-    if (!seoEnabled) return;
+    if (!seoEnabled || !this.el) return;
```

Comment:
This might be interesting to take in anyway, but the gnav is getting closer and closer to get removed and the Eurovision's global-navigation will take over very soon.
So not sure if it's worth a separate PR in those 1-2-3 weeks until that happened.